app/src/main/res/values/strings.xml

```diff
@@ -1,234 +1,489 @@
+         changes the app and the language of the speech-recognition-based search to 'Italian'. -->
+    <string name="settings_language_italian">Italian</string>
+
+    <!-- This string is used to label a button in the 'Settings' dialog window that, when pressed,
+        opens a dialog box that contains display-related settings: window size, pixel density, etc. -->
+    <string name="settings_display">Display</string>
+
+    <!-- This string is used to label a button in the 'Settings' dialog window that, when pressed,
+         opens a dialog box that contains controller-related options: pointer color and scroll direction. -->
+    <string name="settings_controller_options">Controller Options</string>
+
+    <!-- This string is displayed in the 'Settings' dialog window and, is used to label an,
+         On/Off switch that enables/disables the speech-data reporting. -->
+    <string name="settings_report_speech_data">Report Speech Data</string>
+
+    <!-- This string is used to label a button that, when pressed, loads a website
          that allows the user file an issue or problem they may have encountered while using the
          application or while visiting a site. -->
-    <string name="settings_report_issue">Report an issue</string>
-    <!-- This string is used in place of 'settings_version' for developer builds. -->
+    <string name="settings_report_issue">Report an Issue</string>
+
+    <!-- This string is used in place of the 'settings_version' string (e.g., `version 1.1`) when the user is running
+         a developer-build version of the app. -->
     <string name="settings_version_developer">Developer Build</string>
-    <!-- This string is used to label a button in the settings dialog that when pressed
-         opens a dialog box that contains setting an application or Web developer might want to change. -->
-    <string name="settings_developer_options">Developer Options</string>
+
     <!-- This string is the title of a dialog box shown to the user when a settings
          change requires the application to restart. -->
     <string name="restart_dialog_restart">Restart Required</string>
+
     <!-- This string is the text displayed in the restart dialog box explaining why the
-         restart is required. '%1$s' will be replaced at runtime with the app name. -->
+         restart is required. '%1$s' will be replaced at runtime with the website's domain name. -->
     <string name="restart_dialog_text">You must restart %1$s in order to complete changes. Would you like to do that now?</string>
+
     <!-- This string is used to label the button the user presses to restart the
          application in the restart dialog. -->
     <string name="restart_now_dialog_button">Restart Now</string>
-    <!-- This string is used to label the button the user presses to restart the
-     postpone the app restart. -->
+
+    <!-- This string is used to label the button the user presses to restart the postpone the app restart. -->
```

Comment:
I'm not sure I understand that comment

Reply:
Fixed

Reply:
I'm not sure I see a difference in this comment - what I don't understand is what "to restart the postpone the app restart." means